atomic

```diff
@@ -411,6 +415,7 @@ def create_parser(atomic):
     scanp.add_argument("--scan_type", default=None, help=_("define the intended scanner"))
     scanp.add_argument("--list", action='store_true', default=False, help=_("List available scanners"))
     scanp.add_argument("--verbose", action='store_true', default=False, help=_("Show more output from scanning container"))
+    scanp.add_argument("--default_scanner", action='store', default=default_scanner, help=argparse.SUPPRESS)
     scan_group.add_argument("--rootfs", nargs='?', default=[], action='append', help=_("Rootfs path to scan"))
```

Comment:
I am not sure what you are trying to do with this.  I would specify the default_scanner in the config file or force the user to select the scanner they want to use.  I don't see what an option on the command line for default-scanner would do.